packages/cli/src/commands/generate/scaffold/__tests__/scaffold.test.js

```diff
@@ -270,6 +270,20 @@ describe('in javascript (default) mode', () => {
     ).toMatchSnapshot()
   })
 
+  test('error when no editable fields are in model', async () => {
+    expect(
+      async () =>
+        await scaffold.files({
+          ...getDefaultArgs(defaults),
+          model: 'NoEditableField',
+          tests: true,
+          nestScaffoldByModel: true,
+        })
+    ).rejects.toThrow(
+      `There are no editable fields in the NoEditableField model`
+    )
```

Comment:
I slightly updated the handling of promise based exception handling to match the `async/await` option in the docs https://jestjs.io/docs/expect#rejects (and also switched from \` to ')

```suggestion
    await expect(
      scaffold.files({
        ...getDefaultArgs(defaults),
        model: 'NoEditableField',
        tests: true,
        nestScaffoldByModel: true,
      })
    ).rejects.toThrow(
      'There are no editable fields in the NoEditableField model'
    )
```